roofit/histfactory/src/LinInterpVar.cxx

```diff
@@ -100,7 +93,7 @@ LinInterpVar::LinInterpVar(const LinInterpVar& other, const char* name) :
 
 LinInterpVar::~LinInterpVar()
 {
-  delete _paramIter ;
+  //delete _paramIter ;
```

Comment:
```suggestion
```
This commented-out line can be deleted. Actually, the `_paramIter` member is not used anymore now, great!
That means you should remove it also from the class members in `roofit/histfactory/inc/RooStats/HistFactory/LinInterpVar.h`.

Also, the destructor `~LinInterpVar()` does nothing now, so you can remove it as well (also from the header file).